openshift/templates/issuer-web/issuer-web-deploy.yaml

```diff
@@ -68,7 +46,7 @@ objects:
       annotations:
         description: Defines how to deploy the application server
         app.openshift.io/connects-to: >-
-          [{"apiVersion":"apps.openshift.io/v1","kind":"DeploymentConfig","name":"${API_HOST}${SUFFIX}"}]
+          [{"apiVersion":"apps.openshift.io/v1","kind":"DeploymentConfig","name":"${API_HOST}"}]
```

Comment:
This should be the other way around, as per previous comments in the param files.